translations/locales/zh-CN/translations.json

```diff
@@ -0,0 +1,605 @@
+      "MyAssets": "我的资产",
+      "LogOut": "登出"
+    }
+  },
+  "CodemirrorFindAndReplace": {
+    "ToggleReplace": "开关替换选项",
+    "Find": "查找",
+    "FindPlaceholder": "在文件中查找",
+    "Replace": "替换",
+    "ReplaceAll": "替换全部",
+    "ReplacePlaceholder": "替换为",
+    "Regex": "正则表达式",
+    "CaseSensitive": "区分大小写",
+    "WholeWords": "整个单词",
+    "Previous": "上一个",
+    "Next": "下一个",
+    "NoResults": "没有找到",
+    "Close": "关闭"
+  },
+  "LoginForm": {
+    "UsernameOrEmail": "邮箱或用户名",
+    "UsernameOrEmailARIA": "邮箱或用户名",
+    "Password": "密码",
+    "PasswordARIA": "密码",
+    "Submit": "登陆"
+  },
+  "LoginView": {
+    "Title": "p5.js 在线编辑器 ｜ 登陆",
+    "Login": "登陆",
+    "LoginOr": "或",
+    "SignUp": "注册",
+    "Email": "邮箱",
+    "Username": "用户名",
+    "DontHaveAccount": "还没有账号？",
+    "ForgotPassword": "忘记密码？",
+    "ResetPassword": "重置密码"
+  },
+  "SocialAuthButton": {
+    "Connect": "链接 {{serviceauth}} 账号",
+    "Unlink": "取消 {{serviceauth}} 链接",
+    "Login": "使用 {{serviceauth}} 登陆",
+    "LogoARIA": "{{serviceauth}} 图标"
+  },
+  "About": {
+    "Title": "关于",
+    "TitleHelmet": "p5.js 在线编辑器 ｜ 关于",
+    "Contribute": "贡献",
+    "NewP5": "p5.js 新手？",
+    "Report": "报告问题",
+    "Learn": "学习",
+    "Resources": "资源",
+    "Libraries": "第三方库",
+    "Forum": "论坛",
+    "Examples": "范例"
+  },
+  "Toast": {
+    "OpenedNewSketch": "打开了新项目。",
+    "SketchSaved": "项目保存成功。",
+    "SketchFailedSave": "项目保存失败。",
+    "AutosaveEnabled": "自动保存已打开。",
+    "LangChange": "语言已更改",
+    "SettingsSaved": "设置保存成功。"
+  },
+  "Toolbar": {
+    "Preview": "预览",
+    "Auto-refresh": "自动刷新",
+    "OpenPreferencesARIA": "打开设置",
+    "PlaySketchARIA": "运行",
+    "PlayOnlyVisualSketchARIA": "只运行图形项目",
+    "StopSketchARIA": "停止运行",
+    "EditSketchARIA": "修改项目名称",
+    "NewSketchNameARIA": "新项目名称",
+    "By": "作者 "
+  },
+  "Console": {
+    "Title": "控制台",
+    "Clear": "清空",
+    "ClearARIA": "清空控制台",
+    "Close": "关闭",
+    "CloseARIA": "关闭控制台",
+    "Open": "打开",
+    "OpenARIA": "打开控制台"
+  },
+  "Preferences": {
+    "Settings": "设置",
+    "GeneralSettings": "通用",
+    "Accessibility": "包容性设计",
+    "Theme": "主题",
+    "LightTheme": "明亮",
+    "LightThemeARIA": "打开明亮主题",
+    "DarkTheme": "暗黑",
+    "DarkThemeARIA": "打开暗黑主题",
+    "HighContrastTheme": "高对比度",
+    "HighContrastThemeARIA": "打开高对比度主题",
+    "TextSize": "字体大小",
+    "DecreaseFont": "减小",
+    "DecreaseFontARIA": "减小字体大小",
+    "IncreaseFont": "增大",
+    "IncreaseFontARIA": "增大字体大小",
+    "Autosave": "自动保存",
+    "On": "打开",
+    "AutosaveOnARIA": "打开自动保存",
+    "Off": "关闭",
+    "AutosaveOffARIA": "关闭自动保存",
+    "AutocloseBracketsQuotes": "自动添加反括号和反引号",
+    "AutocloseBracketsQuotesOnARIA": "打开自动添加反括号和反引号",
+    "AutocloseBracketsQuotesOffARIA": "关闭自动添加反括号和反引号",
+    "WordWrap": "自动提行",
+    "LineWrapOnARIA": "打开自动提行",
+    "LineWrapOffARIA": "关闭自动提行",
+    "LineNumbers": "行号",
+    "LineNumbersOnARIA": "打开行号",
+    "LineNumbersOffARIA": "关闭行号",
+    "LintWarningSound": "纠码器（Lint）警告音",
```

Comment:
Is there a reason to include "Lint" here? Seeing as below linter are translated as 纠码器 already.

Reply:
I didn't find a universal translation for *Lint* or *Linter* - even Chinese Wiki just refers to it with the English name (https://zh.wikipedia.org/wiki/Lint). So I think it might be helpful to add the original word to avoid confusion. But if you think 纠码器 does well capture its meaning, I'm happy to remove it.

Reply:
For me it does capture the meaning well but I'm not sure if this translation will be recognized as meaning "linter" by native Chinese coding community. The only thing I found that resembles a translation is from an outdated, unofficial translation of the ESLint website (http://eslint.cn/) which translate "linter" as 代码检测工具.

If there's no potential source of confusion by translating it as 纠码器 that you can see, we can stick with this for now.

Reply:
Sure, I will leave 纠码器 for now and make sure it's consistent throughout the translation.